store/bolt/store_test.go

```diff
@@ -0,0 +1,34 @@
+package bolt_test
+
+import (
+	"io/ioutil"
+	"os"
+
+	"github.com/gopheracademy/congo/store/bolt"
+)
+
+// Store represents a test wrapper for bolt.Store.
+type Store struct {
+	*bolt.Store
+}
```

Comment:
This is our test wrapper for `bolt.Store`. Note that the package is `bolt_test` instead of `bolt`. This test wrapper adds helpers to use a random temporary file and it deletes the file on close.